rest-api-spec/src/yamlRestTest/resources/rest-api-spec/test/search.aggregation/40_range.yml

```diff
@@ -11,6 +11,8 @@ setup:
                   type: double
                 long:
                   type: long
+                float:
+                  type: float
```

Comment:
Do you think it's worth doing this for `half_float` too?

Reply:
yes.  I can add that test.

Reply:
thanks!